tools/scripts/generic.mk

```diff
@@ -375,8 +375,8 @@ clean:
 	$(MUTE) $(call remove_file,$(BINARY) $(OBJS) $(ASM_OBJS)) $(HIDE)
 
 # Remove the whole build directory
-PHONY += clean_all
-clean_all: clean_libs
+PHONY += reset
```

Comment:
Is "reset" the correct name here? To me "reset" implies that after the clean operation is done the build starts again. Would "purge" be a better choice?

Reply:
I would say that "purge" is a term that is usual for the removal of sensitive data, while "reset" implies the return to the initial state. However, I would be happy to organize a poll on this matter, and choose the proper naming.

Reply:
No need. Whatever we choose it's gonna be great.